rules/microsoft-365/exfiltration_microsoft_365_pst_exported.toml

```diff
@@ -0,0 +1,44 @@
+[metadata]
+creation_date = "2021/06/06"
+maturity = "production"
+updated_date = "2021/06/06"
+integration = "0365"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies when a user has exported a PST file from Ediscovery search.
+"""
+false_positives = [
+    """
+    PST Export may be performed by a system or network administrator. 
+    """,
+]
+from = "now-30m"
+index = ["filebeat-*", "logs-o365*"]
+language = "kuery"
+license = "Elastic License v2"
+name = "Microsoft 365 Exported PST"
+note = """## Config
+
+The Microsoft 365 Fleet integration, Filebeat module, or similarly structured data is required to be compatible with this rule."""
+
+risk_score = 21
+rule_id = "a3505a1c-ef06-41f7-aca6-66bcf3220fd8"
+severity = "low"
+tags = ["Elastic", "Cloud", "Microsoft 365", "Continuous Monitoring", "SecOps", "Monitoring"]
+timestamp_override = "event.ingested"
+type = "query"
+
+query = '''
+event.dataset:o365.audit and event.provider:Exchange and event.category:web and event.action:"eDiscovery search started or exported" and 
+event.outcome:success
```

Comment:
According to the [docs](https://docs.microsoft.com/en-us/microsoft-365/compliance/alert-policies?view=o365-worldwide), this would be triggered by any of the following:
* A content search is started
* The results of a content search are exported
* A content search report is exported

This is also classified as Informational, we need more context here to make it relevant to merge, like a filter to restrict it to the target activity